### PR TITLE
590 cleanups pt2

### PR DESCRIPTION
**Pull request by luc-tielen**
- [x] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

**Copilot**

### CODE REVIEW

**Comment by trufae** [comment]
Red

**Comment by luc-tielen** [comment]
I think it was red because I didn't do a full rebuild and one of the plugins wasn't migrated. Fixed now. 😄

File: libr/debug/p/debug_winkd.c [hunk 1]
```diff
@@ -108,10 +108,10 @@ static RDebugReasonType r_debug_winkd_wait(RDebug *dbg, int pid) {
 static bool r_debug_winkd_attach(RDebug *dbg, int pid) {
 	RIODesc *desc = dbg->iob.io->desc;
 
-	if (!desc || !desc->plugin || !desc->plugin->name || !desc->data) {
+	if (!desc || !desc->plugin || !desc->plugin->meta.name || !desc->data) {
 		return false;
 	}
-	if (strncmp (desc->plugin->name, "winkd", 6)) {
```
Comment (trufae) [inline libr/debug/p/debug_winkd.c:114]: rstrstartswith()

File: libr/core/cconfig.c [hunk 2]
```diff
@@ -36,7 +36,7 @@ static void set_options(RConfigNode *node, ...) {
 
 static bool isGdbPlugin(RCore *core) {
 	if (core->io && core->io->desc && core->io->desc->plugin) {
-		if (core->io->desc->plugin->name && !strcmp (core->io->desc->plugin->name, "gdb")) {
```
Comment (trufae) [inline libr/core/cconfig.c:39]: better use R_UNWRAP once and reduce later dreferences

**Review by trufae — CHANGES_REQUESTED** [review]

**Pull request merged**